packages/blocks/src/__internal__/utils/types.ts

```diff
@@ -66,7 +66,16 @@ export type BrushMouseMode = {
   lineWidth: number;
 };
 
-export type MouseMode = DefaultMouseMode | ShapeMouseMode | BrushMouseMode;
+export type PanMouseMode = {
+  type: 'pan';
+  temporary?: boolean;
```

Comment:
When will this be false? And having it non-optional seems to make more sense 🤔

Reply:
Moved this property to `EdgelessPageBlock`. The scene is when user enters pan mode by pressing 'Space', we should roll back to the last mouse mode once user releases the key;